pkg/exec/local.go

```diff
@@ -66,9 +69,30 @@ func (cmd *LocalCmd) SetStderr(w io.Writer) Cmd {
 	return cmd
 }
 
-// Run runs
+// Run runs the command
+// If the returned error is non-nil, it should be of type *RunError
 func (cmd *LocalCmd) Run() error {
+	var out bytes.Buffer
+	// TODO(BenTheElder): adding bytes.Buffer to both multiwriters might need
+	// to be wrapped with a mutex
+	if cmd.Stdout != nil {
+		cmd.Stdout = io.MultiWriter(cmd.Stdout, &out)
+	} else {
+		cmd.Stdout = &out
+	}
+	if cmd.Stderr != nil {
+		cmd.Stderr = io.MultiWriter(cmd.Stderr, &out)
+	} else {
+		cmd.Stderr = &out
+	}
 	// TODO: should be in the caller or logger should be injected somehow ...
-	globals.GetLogger().V(3).Infof("Running: %v %v", cmd.Path, cmd.Args)
-	return cmd.Cmd.Run()
+	globals.GetLogger().V(3).Infof("Running: \"%s\"", PrettyCommand(cmd.Args[0], cmd.Args[1:]...))
```

Comment:
why V(3)? what're the criteria?

Reply:
I see it here https://github.com/kubernetes-sigs/kind/pull/847/files#diff-10a41e7a04445626136da682fd56258fR42
it's TRACE level, maybe we should add a comment?

Reply:
There's a TODO to move it entirely.